pygrouper/client.py

```diff
@@ -25,14 +28,24 @@ def _uri(self, endpoint):
 
     def _get(self, endpoint):
         uri = self._uri(endpoint)
-        r = requests.get(uri, auth=(self._api_user, self._api_pass))
-        r.raise_for_status()
+        try:
+            r = requests.get(uri, auth=(self._api_user, self._api_pass), timeout=self._timeout)
+            r.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            raise(GrouperAPIError(err))
+        except requests.exceptions.RequestException as e:
+            raise(GrouperAPIRequestsException(e))
```

Comment:
Pedantic comment - why do we call the exception "err" in one case and "e" in another case?

Reply:
laziness; I noticed that yesterday and "was gonna fix that..."

Reply:
lemme fix